oak_functions/gcp/endpoint_api_config.yaml.template

```diff
@@ -0,0 +1,29 @@
+type: google.api.Service
+config_version: 3
+
+#
+# Name of the service configuration.
+#
+name: ###SERVICE_HOSTNAME###
```

Comment:
For the record, I still find this a bit over-engineered and less readable than having, say, two separate plain YAML configs. I wonder if we could go back to that at some point @conradgrobler ? Is the main issue the unpredictability of the Cloud Run URL?

Reply:
Yes, the main issue is the changing hash in the Cloud Run URL if the instance is recreated. The other issue is that the file will be exactly the same apart from these two values for all deployments. It should be simple to go back to hardcoded config files. We must just then remember to update the files if we recreate the Cloud Run instance.

Reply:
I think hard-coding `SERVICE_HOSTNAME` and `BACKEND_HOSTNAME`, and having to update them manually could be error-prone.